client/extensions/woocommerce/app/settings/email/mailchimp/style.scss

```diff
@@ -105,7 +105,7 @@
 		border-bottom: 1px solid var( --color-neutral-0 );
 		display: flex;
 		align-items: center;
-		background: lighten( $gray-light, 1% );
+		background: var( --color-neutral-0 );
```

Comment:
this one might need some design review. There used to be a slight difference  between the border and the background, but this would remove id.

cc @drw158

Reply:
I think we can merge as is and see if anyone reports an issue.